README.md

```diff
@@ -146,20 +146,23 @@ OpenFeature allows developers to register a provider. Providers are responsible
 using the flag identity and context to determine the state of the feature. If no
 providers are registered, the flags will no-op and return the default value.
 
-### No-op Demo
+### Demo Providers
+
+#### No-op Demo
 
 To see this in action, we'll run the API app found
 [here](./packages/api/src/main.ts). Notice that we're using OpenFeature but not
 registering a provider. Let's run the app and see what happens.
 
 1. Run `npm run no-op-demo`
-2. Open http://localhost:3333/api in your browser
+2. Open http://localhost:3333/message, http://localhost:3333/hex-color/markup, or http://localhost:3333/calculate?num=40 in your browser
+3. Optionally, run the UI as described in the [introductory demo](#introductory-demo)
 
 That's it! You should see **Welcome to the api!**. Unfortunately, that's all you
 can do without registering a provider. Thankfully, as we'll see in the next
 section, that part is easy.
 
-## Compatibility
+#### Compatibility
```

Comment:
Do you want to put this under the Demo Provider section?

Reply:
It was before... so I kept it there. It was in the middle of the providers.